### DISCUSSION TITLE
Features you would like to see?

### DISCUSSION THREAD

**Discussion by cyp3rius** [VirtusLab-Open-Source/strapi-plugin-comments#100] [Ideas]
Hello Folks!

As we're already finished v4 re-implementation, it's time to ask you, the community, what features you're missing the most and would like to see in upcoming releases of the plugin? :) 

We're going to build a community driven backlog combined with our own we're working on right now. Time start!

**Comment by ErwanRaulo** [comment]
The possibility to like a comment. what would be the current alternative ?

**Reply by cyp3rius** [reply]
We are considering 2 options (straightforward VS generic). Both are going to contain likes :)

**Reply by ErwanRaulo** [reply]
Great ! when does one of those options would be ready, Q1 Q2 or later ? i'm looking forward to try it out

**Reply by cyp3rius** [reply]
Beginning of Q2 I think :) but let us first collect community feedback and plan the build

**Reply by kwiat1990** [reply]
To be honest, I did think the previous version of the comment plugin had this feature. It's a must have :)

**Reply by fated-x** [reply]
We finally got around to cutting over our Strapi version from 3 to 4 this week and the only feature we had to drop is "likes". We would love to see those returned. Just wondering if this feature has made its way onto the roadmap for 2023?
